<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1474</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="ui_logedit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>571</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>0</y>
      <width>31</width>
      <height>991</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;操作日志&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>851</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>691</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:700;&quot;&gt;阿尔泰激光设备上位机界面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;串口连接&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>851</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="ui_signallight">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;指示灯&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;轴状态显示：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="ui_tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>831</width>
      <height>192</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <row>
     <property name="text">
      <string>当前轴位置</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>速度</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>加速度</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>轴状态</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Axis1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Axis2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Axis3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Axis4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Axis5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Axis6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Axis7</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Axis8</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>222</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>222</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>22221</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>222233</string>
     </property>
    </item>
    <item row="0" column="5">
     <property name="text">
      <string>222</string>
     </property>
    </item>
    <item row="0" column="6">
     <property name="text">
      <string>1111</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>-1</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>-2</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>停止</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>运动中</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>急停</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>851</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="ui_PortCombox">
       <item>
        <property name="text">
         <string>COM0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM7</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ui_connectbutton">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>连接串口</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; text-decoration: underline;&quot;&gt;闭环控制&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;运动参数设置：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>53</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>460</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; text-decoration: underline;&quot;&gt;开环控制&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>831</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;当前轴位置：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;加速度（P/ms^2）：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;偏移量（P）：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLineEdit" name="ui_AxisPrfPosedit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="ui_Accedit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="ui_OffSetedit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;修改位置（P）：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;减速度（P/ms^2）:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;速度（P/ms）：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <item>
        <widget class="QLineEdit" name="ui_Posedit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="ui_Decedit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="ui_Speededit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>413</y>
      <width>831</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ui_readdatabutton">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>读取数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ui_autocontrolbutton">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>自动控制</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ui_emgstopbutton">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>紧急停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>841</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>16</width>
      <height>141</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>841</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>540</y>
      <width>16</width>
      <height>141</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>831</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="ui_comfirmchangebutton">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>执行修改</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="ui_singelaxisenablebutton">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>当前轴使能</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ui_singleaxiszerobutton">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>当前轴状态清零</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ui_singleaxisstopbutton">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>当前轴停止运动</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="ui_allaxisenablebutton">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>所有轴使能</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ui_allaxiszerobutton">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>全部轴状态清零</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ui_allaxisstopbutton">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>所有轴停止运动</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>831</width>
      <height>49</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;选择轴：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ui_selectaxiscombox">
       <property name="styleSheet">
        <string notr="true">font: 24pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <item>
        <property name="text">
         <string>Axis1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Axis2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Axis3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Axis4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Axis5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Axis6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Axis7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Axis8</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>591</width>
      <height>861</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;数&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;据&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;采&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;集&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;视&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;图&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;AD negative&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QChartView" name="ui_chartView"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;AD positive&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QChartView" name="ui_chartView2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;DA&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QChartView" name="ui_chartView3"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1474</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
